src/validator.js

```diff
@@ -463,7 +460,7 @@ export default class Validator {
 
   _getAddonArchitecture(xpiMetadata) {
     // If we find a file named bootstrap.js this is assumed to be a
-    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011aba8bf8c665aef2b51eb26d0697b3e19c3/validator/testcases/jetpack.py#L154
+    // Jetpack add-on: https://github.com/mozilla/amo-validator/blob/7a8011a/validator/testcases/jetpack.py#L154
```

Comment:
Yeah - this article sums it up quite nicely [1]  - I think in our small repos we should be ok with 7chars. But even if that broke it wouldn't be the end of the world.


[1] http://blog.cuviper.com/2013/11/10/how-short-can-git-abbreviate/